### PR TITLE
Publish site via GitHub Actions

### PR DESCRIPTION
**Pull request by yokra9**
ref: https://github.com/dblock/log4jna/issues/40

### CODE REVIEW

File: .github/workflows/Release.yml [hunk 1]
```diff
@@ -7,33 +7,49 @@ jobs:
...
           server-id: ossrh
           server-username: OSSRH_USERNAME
           server-password: OSSRH_PASSWORD
           gpg-private-key: ${{ secrets.MAVEN_GPG_PRIVATE_KEY }}
           gpg-passphrase: MAVEN_GPG_PASSPHRASE
 
-      # Release
-      - run: mvn clean install -P release 
+      # Set git config
+      - run: |
```
Comment (dblock) [inline .github/workflows/Release.yml:35]: Consider using a GHA for this, I googled https://github.com/OleksiyRudenko/gha-git-credentials not sure if it's the best.

**Review by dblock — COMMENTED** [review]
Great work.

File: .github/workflows/Release.yml (hunk 1 above)
Comment (yokra9) [reply .github/workflows/Release.yml:35]: I tried to use it at https://github.com/dblock/log4jna/pull/55 but failed. It may run only on Linux.

Comment (dblock) [reply .github/workflows/Release.yml:35]: NBD, thanks for trying

**Pull request merged**